dot/rpc/modules/childstate_test.go

```diff
@@ -44,7 +44,44 @@ func createTestTrieState(t *testing.T) (*trie.Trie, common.Hash) {
+	expStr := []string{
+		"0x11f3ba2e1cdd6d62f2ff9b5589e7ff81ba7fb8745735dc3be2a2c61a72c39e78",
```

Comment:
we have `tests/data`. I think places like that could be used to store such static data.

Reply:
This slice is only used by the `TestChildStateModule_GetKeys` and is not such a big data value and I think it is not worth putting it into a file just to load it in just one test function